### PR TITLE
[CognitoIdentityProvider] ResendConfirmationCode

### PR DESCRIPTION
**Pull request by rmikalkenas**

### CODE REVIEW

File: src/Service/CognitoIdentityProvider/tests/Unit/Result/ResendConfirmationCodeResponseTest.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+class ResendConfirmationCodeResponseTest extends TestCase
+{
+    public function testResendConfirmationCodeResponse(): void
+    {
+        // see https://docs.aws.amazon.com/cognitoidentityprovider/latest/APIReference/API_ResendConfirmationCode.html
+        $response = new SimpleMockedResponse(\json_encode([
+            'CodeDeliveryDetails' => [
+                'AttributeName' => 'a',
+                'DeliveryMedium' => 'b',
```
Comment (jderusse) [inline src/Service/CognitoIdentityProvider/tests/Unit/Result/ResendConfirmationCodeResponseTest.php:21]: Coul you please use Enum `DeliveryMediumType`?

Comment (rmikalkenas) [reply src/Service/CognitoIdentityProvider/tests/Unit/Result/ResendConfirmationCodeResponseTest.php]: fixed. By the way, I have noticed that there is a `changelog` file, should I add info about new endpoint to it or is it updated automatically?

Comment (jderusse) [reply src/Service/CognitoIdentityProvider/tests/Unit/Result/ResendConfirmationCodeResponseTest.php]: I'll apreciate it (see how other operation are handled).

Thanks

Comment (rmikalkenas) [reply src/Service/CognitoIdentityProvider/tests/Unit/Result/ResendConfirmationCodeResponseTest.php]: Other PR's are already merged.. Is it ok if I put all the new endpoints to this PR as a separate commit?

Comment (rmikalkenas) [reply src/Service/CognitoIdentityProvider/tests/Unit/Result/ResendConfirmationCodeResponseTest.php]: anyway, pushed as a separate commit. If you think that it has to be as a single commit - say and I will do change

**Comment by jderusse** [comment]
thank you @rmikalkenas for all those contribution. Really appreciated.

**Pull request merged**